modules/ROOT/pages/migration-from-5x.adoc

```diff
@@ -3,88 +3,585 @@
 
-[[plugin-changes]]
-== Plugin Changes
+Previously, two other formats were allowed: comma-separated strings with spaces and arrays of space-separated strings.
 
-=== Tabfocus plugin removed
+Neither of these formats are allowed in {productname} 6.0.
 
-The Tabfocus plugin has been removed in {productname} 6.0. This includes the `+tabfocus_elements+` option.
+=== Removed plugins
 
-Instead, tab order should be configured by specifying the https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex[`+tabindex+` attribute] on the target element for the editor, or the equivalent if using a framework. This allows the tab order to be handled by the browser. In inline mode, this works as in {productname} 5.x. In iframe (classic editor) mode, {productname} now copies the `+tabindex+` attribute from the target element to the editor's iframe, to allow this to work correctly.
+All references to any of these plugins should be removed from plugin configurations.
 
-This allows integrators to define the tab order of the page using standard HTML attributes, without needing to specially configure {productname}.
+NOTE: Leaving references to removed plugins in a plugin configuration will cause console errors documenting the plugin failing to load. {productname} 6.0 should, otherwise, work without error. If, however, {productname} 6.0 is ‘installed’ incorrectly — by being unZipped onto an existing {productname} 5.0 instance for example — the old plugin will not be removed, it will load and, since a {productname} 5.x plugin is attempting to work with a {productname} 6.0 instance, error notifications will almost certainly present to the end-user.
 
-For information on specifying tab order, see: xref:accessibility.adoc#taborder[Tab order].
+See below for specific documentation of potential migration issues, if any, regarding these removed plugins.
 
-[[option-setting-changes]]
-=== Option/Setting changes
+Seven `plugins` were removed because their functionality is now a Core part of {productname} 6.0:
 
-[[renamed-options]]
-==== Renamed Options
+. `colorpicker`
+. `contextmenu`
+. `hr`
+. `noneditable`
+. `paste`
+. `print`
+. `textcolor`
 
-The following options have been renamed in {productname} 6.0:
+Three plugins were removed as per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement]:
 
-* The `+default_link_target+` option has been renamed to `+link_default_target+` for both `+link+` and `+autolink+` plugins.
-* The `+rel_list+` option has been renamed to `+link_rel_list+` for the `+link+` plugin.
-* The `+target_list+` option has been renamed to `+link_target_list+` for the `+link+` plugin.
-* The `+noneditable_editable_class+` option has been renamed to `+editable_class+`.
-* The `+noneditable_noneditable_class+` option has been renamed to `+noneditable_class+`.
+. `fullpage`
+. `bbcode`
+. `legacyoutput`
 
+These plugins were already empty. As of {productname} 6.0, however, they are gone entirely.
```

Comment:
This isn't correct, I think. Colourpicker, contextmenu and... something else were mock plugins, but these still existed in T5. @lnewson please sanity check?

Reply:
Yeah that's not correct. The following were "empty" plugins in 5.x:
- contextmenu
- colorpicker
- textcolor
then the following worked in 5.x, but have now been removed in 6.0:
- bbcode
- fullpage
- legacyoutput
- spellchecker

Reply:
And then there's also tabfocus, which has been removed in 6 and there is an alternate way (using standard HTML tricks) to get the same behaviour

Reply:
I noticed this information is all correct in the http://docs-feature-6-doc-1559.staging.tiny.cloud/docs/tinymce/6/6.0-release-notes-plugins/ page, if that helps